source/agora/consensus/EnrollmentManager.d

```diff
@@ -722,19 +722,18 @@ public class EnrollmentManager
         Get validator's pre-image from the validator set.
 
         Params:
-            enroll_key = The key for the enrollment in which the pre-image is
-                contained.
+            utxo = The frozen UTXO used for enrollment.
 
         Returns:
             the PreImageInfo of the enrolled key if it exists,
             otherwise PreImageInfo.init
 
     ***************************************************************************/
 
-    public PreImageInfo getValidatorPreimage (in Hash enroll_key)
-        @trusted nothrow
+    public PreImageInfo getValidatorPreimage (in Hash utxo)
+        @safe nothrow
```

Comment:
`@trusted` presents an interface as `@safe` without doing the checks.
If the function is a trivial wrapper for another function, it's likely that the `@trusted` should be on that other function.

For example, the following is fine:
```D
c{
    void* ret = malloc(T.sizeof);
    initializeMemory(ret);
    return cast(T*) ret;
}
```
Because it wraps a `@system` function in a safe manner.
However, this is not:
```D
void* trustedMalloc (size_t size) @trusted { return malloc(size); }
```
Because if you can write this, it means `malloc` should be safe to call directly too (after all `trustedMalloc` only applies attributes, nothing else), which it clearly is not.

Reply:
Ty.